micromasters/models.py

```diff
@@ -27,7 +28,7 @@ def update(self, **kwargs):
         database level without loading objects into memory.
         """
         if "updated_on" not in kwargs:
-            kwargs["updated_on"] = datetime.datetime.utcnow()
+            kwargs["updated_on"] = datetime.datetime.now(tz=pytz.UTC)
```

Comment:
That's populated when the user creates the object in question. The database will put a UTC time zone on the date because of the `USE_TZ` setting so we don't need to worry about it there